### PR TITLE
Cut seconds and ms off of time stamp

### PR DESCRIPTION
**Pull request by noramyer**

### CODE REVIEW

File: src/codeu/chat/client/simplegui2/ChatMenu.java [hunk 1]
```diff
@@ -298,8 +299,12 @@ private void getAllMessages(ConversationSummary conversation) {
       // Display author name if available.  Otherwise display the author UUID.
       final String authorName = clientContext.user.getName(m.author);
 
+      Time time= m.creation;
+      String cutSecondsOffTime= time.toString();
+      cutSecondsOffTime=cutSecondsOffTime.substring(0, cutSecondsOffTime.length()- 7);
```
Comment (rshar1) [inline src/codeu/chat/client/simplegui2/ChatMenu.java]: Interesting. However, the string might not always be the same length. For example, what if it was 0 milliseconds. Would it display 000 or 0? We should check on that first.

**Review by rshar1 — CHANGES_REQUESTED** [review]
If you look into the Time class, the time is stored as type Date. I think there is a way to modify the date to only return certain properties, such as Hours and minutes. You can probably also specify 12 hour clock vs 24 hour clock. We should read up on the date class for this

File: src/codeu/chat/client/simplegui2/ChatMenu.java (hunk 1 above)
Comment (noramyer) [reply src/codeu/chat/client/simplegui2/ChatMenu.java]: Good point I thought about that

**Review by rshar1 — APPROVED** [review]
Looks good

**Pull request merged**